frontend/src/mainApp.js

```diff
@@ -25,6 +25,10 @@ function App() {
         <Route exact path='/CreateFolder' component={CreateFolder} />
         <Route exact path='/InsideFolder' component={InsideFolder} />
         <Route exact path='/SimilarWords' component={SimilarWords} />
+        <Route exact path='/homePage' component={() => {
```

Comment:
Can you also add just the root "/" to go to the homepage?

Reply:
This is for the logout button inside of the mainApp. React Servlet is making sure that if the user types in the root then they will be rerouted to the homePage.